scripts/bench/tts-bench.py

```
@@ -0,0 +1,650 @@
+        self.python_bin = python_bin
+        self.model_dir = model_dir
+        self.proc: subprocess.Popen | None = None
+        self.load_seconds = 0.0
+        self.device = "unknown"
+        self.peak_rss_kb = 0
+
+    def _start(self) -> subprocess.Popen:
+        if self.proc is not None:
+            return self.proc
+        start = time.monotonic()
+        proc = subprocess.Popen(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)